provisioning/device/src/Utilities/Argument.cs

```diff
@@ -53,19 +53,20 @@ public static void AssertNotNullOrWhiteSpace(string value, string name)
         /// <summary>
         /// Checks an argument to ensure that its value isn't negative.
         /// </summary>
-        /// <param name="argumentValue">The value of the argument.</param>
-        /// <param name="argumentName">The name of the argument for diagnostic purposes.</param>
-        internal static void AssertNotNegativeValue<T>(T argumentValue, string argumentName)
+        /// <param name="value">The value of the argument.</param>
+        /// <param name="name">The name of the argument for diagnostic purposes.</param>
+        /// <exception cref="ArgumentOutOfRangeException"><paramref name="value"/> is negative.</exception>
+        internal static void AssertNotNegativeValue<T>(T value, string name)
```

Comment:
I like what you've changed it to but I'm curious - did the tool highlight this change or was it personal preference?

Reply:
It would be great if the tools are capable to highlight this, but unfortunately no, I had to manually align and update it.